Error while checking build/O2/fullCI_slc9 for 0ae50be0f106dcffa79a78207d21debb0c2dd69c at 2026-03-05 17:10:
```
## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:669:146: error: use nullptr [modernize-use-nullptr]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:712:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:722:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:736:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:745:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:769:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:777:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:791:46: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/15122-slc9_x86-64/0/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx:799:46: error: statement should be inside braces [readability-braces-around-statements]
++ [[ 0 == 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/15122/0ae50be0f106dcffa79a78207d21debb0c2dd69c/build_O2_fullCI_slc9/pretty.html).

--- a/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
+++ b/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
@@ -770,16 +770,18 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
 
       double rCur = rMinMiddleServicesFwd;
       double dR = siO2FiberAreaD / (3.14 * rCur);
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dR += siO2FiberAreaB / (3.14 * rCur);
+      }
       TGeoTubeSeg* middleFwdFiberSIO2 = new TGeoTubeSeg(Form("TRK_MIDFWD_FIBER_SIO2sh_%s%d", orLabel.c_str(), iSide), rCur, rCur + dR, zLengthMiddleServicesFwd / 2, -45, 45);
       TGeoVolume* middleFwdFiberSIO2Volume = new TGeoVolume(Form("TRK_MIDFWD_FIBER_SIO2_%s%d", orLabel.c_str(), iSide), middleFwdFiberSIO2, medSiO2);
       middleFwdFiberSIO2Volume->SetLineColor(kGray);
 
       rCur += dR;
       dR = peFiberAreaD / (3.14 * rCur);
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dR += peFiberAreaB / (3.14 * rCur);
+      }
       TGeoTubeSeg* middleFwdFiberPE = new TGeoTubeSeg(Form("TRK_MIDFWD_FIBER_PEsh_%s%d", orLabel.c_str(), iSide), rCur, rCur + dR, zLengthMiddleServicesFwd / 2, -45, 45);
       TGeoVolume* middleFwdFiberPEVolume = new TGeoVolume(Form("TRK_MIDFWD_FIBER_PE_%s%d", orLabel.c_str(), iSide), middleFwdFiberPE, medPE);
       middleFwdFiberPEVolume->SetLineColor(kGray);
@@ -792,16 +794,18 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
       // Create powerlines
       rCur += dR;
       dR = cuPowerAreaD / (3.14 * rCur);
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dR += cuPowerAreaB / (3.14 * rCur);
+      }
       TGeoTubeSeg* middleFwdPowerCu = new TGeoTubeSeg(Form("TRK_MIDFWD_POWER_CUsh_%s%d", orLabel.c_str(), iSide), rCur, rCur + dR, zLengthMiddleServicesFwd / 2, -45, 45);
       TGeoVolume* middleFwdPowerCuVolume = new TGeoVolume(Form("TRK_MIDFWD_POWER_CU_%s%d", orLabel.c_str(), iSide), middleFwdPowerCu, medCu);
       middleFwdPowerCuVolume->SetLineColor(kGray);
 
       rCur += dR;
       dR = pePowerAreaD / (3.14 * rCur);
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dR += pePowerAreaB / (3.14 * rCur);
+      }
       TGeoTubeSeg* middleFwdPowerPE = new TGeoTubeSeg(Form("TRK_MIDFWD_POWER_PEsh_%s%d", orLabel.c_str(), iSide), rCur, rCur + dR, zLengthMiddleServicesFwd / 2, -45, 45);
       TGeoVolume* middleFwdPowerPEVolume = new TGeoVolume(Form("TRK_MIDFWD_POWER_PE_%s%d", orLabel.c_str(), iSide), middleFwdPowerPE, medPE);
       middleFwdPowerPEVolume->SetLineColor(kGray);

--- a/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
+++ b/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
@@ -331,10 +331,10 @@ void TRKServices::createMiddleServices(TGeoVolume* motherVolume)
   const float rMaxMiddleBarrelDisk = 35.f;
   const float zLengthMiddleBarrel = 64.5f;
   for (auto& orientation : {Orientation::kASide, Orientation::kCSide}) {
-    TGeoTube* middleBarrelConnDiskSIO2 = new TGeoTube(Form("TRK_MIDBARCONN_DISK_SIO2sh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, siO2FiberThick / 2.);
-    TGeoTube* middleBarrelConnDiskPE = new TGeoTube(Form("TRK_MIDBARCONN_DISK_PEsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, peFiberThick / 2.);
-    TGeoVolume* middleBarrelConnDiskSIO2Volume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_SIO2_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskSIO2, medSiO2);
-    TGeoVolume* middleBarrelConnDiskPEVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_PE_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskPE, medPE);
+    TGeoTube* middleBarrelConnDiskSIO2 = new TGeoTube(Form("TRK_MIDBARCONN_DISK_FIBER_SIO2sh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, siO2FiberThick / 2.);
+    TGeoTube* middleBarrelConnDiskPE = new TGeoTube(Form("TRK_MIDBARCONN_DISK_FIBER_PEsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, peFiberThick / 2.);
+    TGeoVolume* middleBarrelConnDiskSIO2Volume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_FIBER_SIO2_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskSIO2, medSiO2);
+    TGeoVolume* middleBarrelConnDiskPEVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_FIBER_PE_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskPE, medPE);
     middleBarrelConnDiskSIO2Volume->SetLineColor(kGray);
     middleBarrelConnDiskPEVolume->SetLineColor(kGray);
     auto* rot = new TGeoRotation("", 0, 0, 180);
@@ -343,10 +343,10 @@ void TRKServices::createMiddleServices(TGeoVolume* motherVolume)
     motherVolume->AddNode(middleBarrelConnDiskSIO2Volume, 1, combiTransSIO2);
     motherVolume->AddNode(middleBarrelConnDiskPEVolume, 1, combiTransPE);
 
-    TGeoTube* middleBarrelConnDiskCu = new TGeoTube(Form("TRK_MIDBARCONN_DISK_CUsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, cuPowerThick / 2.);
-    TGeoTube* middleBarrelConnDiskPEPower = new TGeoTube(Form("TRK_MIDBARCONN_DISK_PEsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, pePowerThick / 2.);
-    TGeoVolume* middleBarrelConnDiskCuVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_CU_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskCu, medCu);
-    TGeoVolume* middleBarrelConnDiskPEPowerVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_PE_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskPEPower, medPE);
+    TGeoTube* middleBarrelConnDiskCu = new TGeoTube(Form("TRK_MIDBARCONN_DISK_POWER_CUsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, cuPowerThick / 2.);
+    TGeoTube* middleBarrelConnDiskPEPower = new TGeoTube(Form("TRK_MIDBARCONN_DISK_POWER_PEsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, pePowerThick / 2.);
+    TGeoVolume* middleBarrelConnDiskCuVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_POWER_CU_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskCu, medCu);
+    TGeoVolume* middleBarrelConnDiskPEPowerVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_POWER_PE_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskPEPower, medPE);
     middleBarrelConnDiskCuVolume->SetLineColor(kGray);
     middleBarrelConnDiskPEPowerVolume->SetLineColor(kGray);
     auto* combiTransCu = new TGeoCombiTrans(0, 0, (int)orientation * (siO2FiberThick + peFiberThick + cuPowerThick / 2. + zLengthMiddleBarrel), rot);
@@ -354,14 +354,16 @@ void TRKServices::createMiddleServices(TGeoVolume* motherVolume)
     motherVolume->AddNode(middleBarrelConnDiskCuVolume, 1, combiTransCu);
     motherVolume->AddNode(middleBarrelConnDiskPEPowerVolume, 1, combiTransPEPower);
 
-    TGeoTube* middleBarrelConnDiskPU = new TGeoTube(Form("TRK_MIDBARCONN_DISK_PUsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, puCoolingThick);
-    TGeoTube* middleBarrelConnDiskH2O = new TGeoTube(Form("TRK_MIDBARCONN_DISK_H2Osh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, h2oCoolingThick);
+    TGeoTube* middleBarrelConnDiskPU = new TGeoTube(Form("TRK_MIDBARCONN_DISK_PUsh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, puCoolingThick / 2.);
+    TGeoTube* middleBarrelConnDiskH2O = new TGeoTube(Form("TRK_MIDBARCONN_DISK_H2Osh_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, h2oCoolingThick / 2.);
     TGeoVolume* middleBarrelConnDiskPUVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_PU_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskPU, medPU);
     TGeoVolume* middleBarrelConnDiskH2OVolume = new TGeoVolume(Form("TRK_MIDBARCONN_DISK_H2O_%s", orientation == Orientation::kASide ? "bwd" : "fwd"), middleBarrelConnDiskH2O, medH2O);
     middleBarrelConnDiskPUVolume->SetLineColor(kGray);
     middleBarrelConnDiskH2OVolume->SetLineColor(kGray);
-    motherVolume->AddNode(middleBarrelConnDiskPUVolume, 1, combiTransCu);
-    motherVolume->AddNode(middleBarrelConnDiskH2OVolume, 1, combiTransPEPower);
+    auto* combiTransPU = new TGeoCombiTrans(0, 0, (int)orientation * (siO2FiberThick + peFiberThick + cuPowerThick + pePowerThick + puCoolingThick / 2. + zLengthMiddleBarrel), rot);
+    auto* combiTransH2O = new TGeoCombiTrans(0, 0, (int)orientation * (siO2FiberThick + peFiberThick + cuPowerThick + pePowerThick + puCoolingThick + h2oCoolingThick / 2. + zLengthMiddleBarrel), rot);
+    motherVolume->AddNode(middleBarrelConnDiskPUVolume, 1, combiTransPU);
+    motherVolume->AddNode(middleBarrelConnDiskH2OVolume, 1, combiTransH2O);
   }
 
   // Barrel to forward connection disks
@@ -533,7 +535,7 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
   const int ITBarrelnPower = 70;
   float siO2FiberAreaB = ITBarrelnFiber * mFiberArea * mFiberComposition[0];
   float peFiberAreaB = ITBarrelnFiber * mFiberArea * mFiberComposition[1];
-  ;
+
   float puCoolingAreaB = 0;
   float h2oCoolingAreaB = 0;
   float cuPowerAreaB = ITBarrelnPower * mPowerBundleArea * mPowerBundleComposition[0];
@@ -544,7 +546,7 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
   const int ITDisknPower = 3 * 16;
   float siO2FiberAreaD = ITDisknFiber * mFiberArea * mFiberComposition[0];
   float peFiberAreaD = ITDisknFiber * mFiberArea * mFiberComposition[1];
-  ;
+
   float puCoolingAreaD = 0;
   float h2oCoolingAreaD = 0;
   float cuPowerAreaD = ITDisknPower * mPowerBundleArea * mPowerBundleComposition[0];
@@ -579,16 +581,16 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
 
   double zCur = zLengthMiddleBarrel;
   double dZ = siO2FiberAreaB / diskCircumference / 2.;
-  TGeoTube* middleBarrelConnDiskSIO2 = new TGeoTube("TRK_MIDBARCONN_DISK_SIO2sh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
-  TGeoVolume* middleBarrelConnDiskSIO2Volume = new TGeoVolume("TRK_MIDBARCONN_DISK_SIO2", middleBarrelConnDiskSIO2, medSiO2);
+  TGeoTube* middleBarrelConnDiskSIO2 = new TGeoTube("TRK_MIDBARCONN_DISK_FIBER_SIO2sh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
+  TGeoVolume* middleBarrelConnDiskSIO2Volume = new TGeoVolume("TRK_MIDBARCONN_DISK_FIBER_SIO2", middleBarrelConnDiskSIO2, medSiO2);
   middleBarrelConnDiskSIO2Volume->SetLineColor(kGray);
   auto* rot = new TGeoRotation("", 0, 0, 180); // Why this?
   auto* combiTransSIO2 = new TGeoCombiTrans(0, 0, (int)orientation * (zCur + dZ), rot);
 
   zCur += 2. * dZ;
   dZ = peFiberAreaB / diskCircumference / 2.;
-  TGeoTube* middleBarrelConnDiskPE = new TGeoTube("TRK_MIDBARCONN_DISK_PEsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
-  TGeoVolume* middleBarrelConnDiskPEVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_PE", middleBarrelConnDiskPE, medPE);
+  TGeoTube* middleBarrelConnDiskPE = new TGeoTube("TRK_MIDBARCONN_DISK_FIBER_PEsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
+  TGeoVolume* middleBarrelConnDiskPEVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_FIBER_PE", middleBarrelConnDiskPE, medPE);
   middleBarrelConnDiskPEVolume->SetLineColor(kGray);
   auto* combiTransPE = new TGeoCombiTrans(0, 0, (int)orientation * (zCur + dZ), rot);
 
@@ -597,15 +599,15 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
 
   zCur += 2. * dZ;
   dZ = cuPowerAreaB / diskCircumference / 2.;
-  TGeoTube* middleBarrelConnDiskCu = new TGeoTube("TRK_MIDBARCONN_DISK_CUsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
-  TGeoVolume* middleBarrelConnDiskCuVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_CU", middleBarrelConnDiskCu, medCu);
+  TGeoTube* middleBarrelConnDiskCu = new TGeoTube("TRK_MIDBARCONN_DISK_POWER_CUsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
+  TGeoVolume* middleBarrelConnDiskCuVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_POWER_CU", middleBarrelConnDiskCu, medCu);
   middleBarrelConnDiskCuVolume->SetLineColor(kGray);
   auto* combiTransCu = new TGeoCombiTrans(0, 0, (int)orientation * (zCur + dZ), rot);
 
   zCur += 2. * dZ;
   dZ = pePowerAreaB / diskCircumference / 2.;
-  TGeoTube* middleBarrelConnDiskPEPower = new TGeoTube("TRK_MIDBARCONN_DISK_PEsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
-  TGeoVolume* middleBarrelConnDiskPEPowerVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_PE", middleBarrelConnDiskPEPower, medPE);
+  TGeoTube* middleBarrelConnDiskPEPower = new TGeoTube("TRK_MIDBARCONN_DISK_POWER_PEsh", rMinMiddleBarrelDisk, rMaxMiddleBarrelDisk, dZ);
+  TGeoVolume* middleBarrelConnDiskPEPowerVolume = new TGeoVolume("TRK_MIDBARCONN_DISK_POWER_PE", middleBarrelConnDiskPEPower, medPE);
   middleBarrelConnDiskPEPowerVolume->SetLineColor(kGray);
   auto* combiTransPEPower = new TGeoCombiTrans(0, 0, (int)orientation * (zCur + dZ), rot);
   motherVolume->AddNode(middleBarrelConnDiskCuVolume, 1, combiTransCu);

--- a/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
+++ b/Detectors/Upgrades/ALICE3/TRK/simulation/src/TRKServices.cxx
@@ -666,7 +666,7 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
         TGeoTubeSeg* middleBarrelFiberPE = new TGeoTubeSeg(Form("TRK_MLB_FIBER_PEsh_A%d", iSide), rCur, rCur + dR, zLengthCylinderMiddleServices, -45, 45);
         TGeoVolume* middleBarrelFiberPEVolume = new TGeoVolume(Form("TRK_MLB_FIBER_PE_A%d", iSide), middleBarrelFiberPE, medPE);
         middleBarrelFiberPEVolume->SetLineColor(kGray);
-        auto* combiTrans = new TGeoCombiTrans(0, 0, (int)orientation * (zLengthMiddleServices - zLengthCylinderMiddleServices), new TGeoRotation(0, refAngle + iSide * 180., 0, 0));
+        auto* combiTrans = new TGeoCombiTrans(0, 0, (int)orientation * (zLengthMiddleServices - zLengthCylinderMiddleServices), new TGeoRotation(nullptr, refAngle + iSide * 180., 0, 0));
         motherVolume->AddNode(middleBarrelFiberSIO2Volume, 1, combiTrans);
         motherVolume->AddNode(middleBarrelFiberPEVolume, 1, combiTrans);
 
@@ -709,8 +709,9 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
       double zCur = zLengthMiddleServices; // Change to f
       double dZ = siO2FiberAreaD / diskCircumference / 2.;
       totalThickness += 2 * dZ;
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dZ += siO2FiberAreaB / diskCircumference / 2.;
+      }
       TGeoTubeSeg* middleBarFwdFiberSIO2 = new TGeoTubeSeg(Form("TRK_MIDBARFWD_FIBER_SIO2sh_%s%d", orLabel.c_str(), iSide), rMinMiddleBarrel, rMaxMiddleServicesBarFwd, dZ, -45, 45);
       TGeoVolume* middleBarFwdFiberSIO2Volume = new TGeoVolume(Form("TRK_MIDBARFWD_FIBER_SIO2_%s%d", orLabel.c_str(), iSide), middleBarFwdFiberSIO2, medSiO2);
       auto* rot = new TGeoRotation("", refAngle + iSide * 180., 0, 180.);
@@ -719,8 +720,9 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
       zCur += 2 * dZ;
       dZ = peFiberAreaD / diskCircumference / 2.;
       totalThickness += 2 * dZ;
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dZ += peFiberAreaB / diskCircumference / 2.;
+      }
       TGeoTubeSeg* middleBarFwdFiberPE = new TGeoTubeSeg(Form("TRK_MIDBARFWD_FIBER_PEsh_%s%d", orLabel.c_str(), iSide), rMinMiddleBarrel, rMaxMiddleServicesBarFwd, dZ, -45, 45);
       TGeoVolume* middleBarFwdFiberPEVolume = new TGeoVolume(Form("TRK_MIDBARFWD_FIBER_PE_%s%d", orLabel.c_str(), iSide), middleBarFwdFiberPE, medPE);
       middleBarFwdFiberSIO2Volume->SetLineColor(kGray);
@@ -733,8 +735,9 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
       zCur += 2 * dZ;
       dZ = cuPowerAreaD / diskCircumference / 2.;
       totalThickness += 2 * dZ;
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dZ += cuPowerAreaB / diskCircumference / 2.;
+      }
       TGeoTubeSeg* middleBarFwdPowerCu = new TGeoTubeSeg(Form("TRK_MIDBARFWD_POWER_CUsh_%s%d", orLabel.c_str(), iSide), rMinMiddleBarrel, rMaxMiddleServicesBarFwd, dZ, -45, 45);
       TGeoVolume* middleBarFwdPowerCuVolume = new TGeoVolume(Form("TRK_MIDBARFWD_POWER_CU_%s%d", orLabel.c_str(), iSide), middleBarFwdPowerCu, medCu);
       auto* combiTransCu = new TGeoCombiTrans(0, 0, (int)orientation * (zCur + dZ), rot);
@@ -742,8 +745,9 @@ void TRKServices::createMLServicesPeacock(TGeoVolume* motherVolume)
       zCur += 2 * dZ;
       dZ = pePowerAreaD / diskCircumference / 2.;
       totalThickness += 2 * dZ;
-      if (orientation == Orientation::kASide)
+      if (orientation == Orientation::kASide) {
         dZ += pePowerAreaB / diskCircumference / 2.;
+      }
       TGeoTubeSeg* middleBarFwdPowerPE = new TGeoTubeSeg(Form("TRK_MIDBARFWD_POWER_PEsh_%s%d", orLabel.c_str(), iSide), rMinMiddleBarrel, rMaxMiddleServicesBarFwd, dZ, -45, 45);
       TGeoVolume* middleBarFwdPowerPEVolume = new TGeoVolume(Form("TRK_MIDBARFWD_POWER_PE_%s%d", orLabel.c_str(), iSide), middleBarFwdPowerPE, medPE);
       middleBarFwdPowerCuVolume->SetLineColor(kGray);

